CommonLibSF/vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "commonlibsf",
+  "version-semver": "1.0.0",
```

Comment:
It's worth considering if semver is the right model for this project. Semver assumes some kind of controlled release, which might not be the typically useful one. Perhaps date-based versioning is better, with nightly releases?

Reply:
I like semver more than date since you know if there's a breaking change just by looking. Given this is a dependency, I think that information is necessary.

Reply:
I have not changed the versioning yet, only updated the outdate link of `vcpkg.schema.json`. Please let me know if we decide on a versioning schema for this.

Reply:
Charmed has a point, automated date-based releases make sense here imo, esp early on when there will likely be frequent changes as new things get RE'd

Reply:
Semver already supports date meta data if we really care. https://semver.org/#spec-item-10.

Reply:
The problem with semver for tracking breaking changes is almost every change will be breaking. Any change of an unk member to a named one will be breaking, so almost all changes end up being major version changes anyway. And it will become a shared responsibility to manage that version number with every commit.

Reply:
For comparison, in clib-ng I didn't enforce strict semver semantics. Rather I let RE changes be exempt from the major version change requirements. Which may be fine here. But also it raises the question of who makes the judgement of the version number bump, and who decides when to release? In a shared project that can't be a single person's arbitrary decision as I did in clib-ng.

Reply:
> In a shared project that can't be a single person's arbitrary decision as I did in clib-ng.
I second this, we should let a workflow action to auto bump date-based versioning on PRs

Reply:
Date-based release seems much less of a headache, don't see any reason to not use that (or any reason to prefer semver really)

Reply:
> The problem with semver for tracking breaking changes is almost every change will be breaking. Any change of an unk member to a named one will be breaking, so almost all changes end up being major version changes anyway. And it will become a shared responsibility to manage that version number with every commit.

I could buy this argument. My thought on breaking would be changing something that is already defined. Changing unk (which should be unused and unsupported behavior IMHO) and thus not breaking. But since it looks like the tide wants to use dates, that's fine.

Regardless, version should never be touched in person unless CI break. CI should handle updating versioning.

Reply:
Going to mark this resolved for now. I think before builds start getting published a version strategy should be finalized.